packages/orbit-components/src/Loading/__examples__/TYPES.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import * as React from "react";
+
+import Button from "../../Button";
+import Card, { CardSection } from "../../Card";
+import Heading from "../../Heading";
+import Loading from "../index";
+import Stack from "../../Stack";
+import Text from "../../Text";
+
+export default {
+  Example: () => (
+    <Stack>
+      <Stack spacing="extraTight">
+        <Heading as="h3" type="title3">
+          Page loader
+        </Heading>
+        <Text>Useful when an entire page is loading.</Text>
+        <Loading />
+      </Stack>
+      <Stack spacing="extraTight">
+        <Heading as="h3" type="title3">
+          Box loader
+        </Heading>
+        <Text>Useful when content in boxes like cards and card sections is being loaded.</Text>
+        <Card loading title="Interesting facts">
+          <CardSection>Interesting facts about travel</CardSection>
+        </Card>
+        <Card title="Interesting facts">
+          <CardSection>
+            <Loading type="boxLoader" loading>
+              Interesting facts about travel
+            </Loading>
+          </CardSection>
+        </Card>
+      </Stack>
+      <Stack spacing="extraTight">
+        <Heading as="h3" type="title3">
+          Button loader
+        </Heading>
+        <Text>
+          Useful when a user has interacted with a button to show that something is happening.
+        </Text>
+        <Button loading>Save</Button>
+        <Button iconLeft={<Loading type="buttonLoader" />} disabled>
```

Comment:
I'm not sure @CollierCZ if this is a right use case. It would be if we would not have `loading` boolean property on the Button component.

Reply:
OK. So there's no case to use `buttonLoader` directly then? Thought there might be.